src/openassetio-core/src/hostApi/Manager.cpp

```diff
@@ -46,9 +47,32 @@ ManagerPtr Manager::make(managerApi::ManagerInterfacePtr managerInterface,
       new Manager(std::move(managerInterface), std::move(hostSession)));
 }
 
+namespace {
+/**
+ * Extract the entity reference prefix from a manager plugin's info
+ * dictionary, if available.
+ */
+std::optional<Str> entityReferencePrefixFromInfo(const InfoDictionary &info) {
+  // Check if the info dict has the prefix key.
+  if (auto iter = info.find(Str{constants::kInfoKey_EntityReferencesMatchPrefix});
+      iter != info.end()) {
+    // Extract prefix string, and fail silently if the wrong type is
```

Comment:
[consider] Worth a `kDebugApi` method if the type is wrong?

Reply:
Added (force-pushed) - though did a `warning` rather than debug log.